codespell_lib/_codespell.py

```diff
@@ -50,6 +50,10 @@
 )
 _builtin_default = 'clear,rare'
 
+# docs say os.EX_USAGE is only available on Unix systems, so to be safe we
+# protect and just use the value it is on macOS and Linux (64)
+EX_USAGE = 64
```

Comment:
Also:

```python
EX_OK = 0
EX_DATAERR = 65
```

Reply:
Would it be safer and clearer to use something that doesn't clash with the os definition, although it's a different namespace, e.g. EXIT_USAGE?

Reply:
I think the comment is clear enough, safety wise we can count on this not to change in `os`, and it's arguably clearer to use the name `EX_USAGE` than anything else since that's what we're trying to use

Reply:
Okay.